java-score/src/main/java/com/iconloop/score/pds/PdsPolicy.java

```diff
@@ -486,5 +529,5 @@ public void fallback() {
      * Events
      */
     @EventLog
-    protected void PDSEvent(String event, String value1, String value2) {}
+    protected void PDSEvent(String event, String value1, String value2, int nonce) {}
```

Comment:
```suggestion
    protected void PDSEvent(String event, String value1, String value2, BigInteger nonce) {}
```
Because `nonce` is now a BigInteger type internally, emitting it should also be a BigInteger.